cmd/network.go

```diff
@@ -275,9 +277,68 @@ func generateNetworkPolicies(profile *kubeflowv1.Profile) []*networkingv1.Networ
 		},
 	})
 
+	// Allow ingress to SQL Server from all namespaces
+	policies = append(policies, &networkingv1.NetworkPolicy{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "allow-sql-ingress",
+			Namespace: profile.Name,
+			OwnerReferences: []metav1.OwnerReference{
+				*metav1.NewControllerRef(profile, kubeflowv1.SchemeGroupVersion.WithKind("Profile")),
+			},
+		},
+		Spec: networkingv1.NetworkPolicySpec{
+			PodSelector: metav1.LabelSelector{}, // Apply to all pods in the namespace
```

Comment:
should only apply to notebooks.

Reply:
I defined a PodSelector near the top of the file and now I've referenced the PodSelector in each relevant policy.